### PR TITLE
Docs: Replace Python SDK with new High Level SDK

### LINKED ISSUE #7120: Python Wrapper: Modify Python docs in lakefs.io
Adapt documentation to the new High Level SDK

### PR DESCRIPTION
**Pull request by N-o-Z**
### SHOULD NOT BE MERGED BEFORE OFFICIAL RELEASE!

Closes #7120

### CODE REVIEW

File: docs/integrations/python.md [hunk 1]
```diff
@@ -13,15 +13,18 @@ redirect_from:
...
 
-**Improved Python SDK**  <span class="badge mr-1">New</span><span class="badge">beta</span>  
-Check-out the newly released [Python SDK library](https://pydocs-lakefs.lakefs.io/) - providing simpler interface and improved user experience!  
-*OR* continue reading for the current Python SDK.
+**High Level Python SDK**  <span class="badge mr-1">New</span>  
+We've just released a new High Level Python SDK library, and we're super excited to tell you about it! Continue reading to get the
+full story!  
+**For previous Python SDKs follow these links:**  
+[lakefs-sdk](https://pydocs-sdk.lakefs.io)  
+[legacy-sdk](https://pydocs.lakefs.io)
```
Comment (nopcoder) [inline docs/integrations/python.md:21]: Need to understand that the python sdk is still supported and used by the high level one, don't want people think we deprecate it as we did for the legacy.

Maybe my issue is the two primary ways we describe - as this package uses the sdk itself.

File: docs/integrations/python.md [hunk 2]
```diff
@@ -31,119 +34,89 @@ Install the Python client using pip:
...
-This setting allows well-known "man-in-the-middle",
-impersonation, and credential stealing attacks.  Never use this in any
-production setting.
-
-Optionally, to enable communication via proxies, simply set the proxy configuration:
-
-```python
-configuration.ssl_ca_cert = <path to a file of concatenated CA certificates in PEM format> # Set this to customize the certificate file to verify the peer
-configuration.proxy = <proxy server URL>
-```
```
Comment (nopcoder) [inline docs/integrations/python.md:69]: Don't think we need to remove this part - it was added after iddo had to resolve issues with users that had issues working with custom certificates and web proxy.

**Review by nopcoder — CHANGES_REQUESTED** [review]

File: docs/integrations/python.md (hunk 1 above)
Comment (N-o-Z) [reply docs/integrations/python.md:21]: > and used by the high level one

It's an implementation detail that the user shouldn't care about

File: docs/integrations/python.md [hunk 3]
```diff
@@ -55,6 +56,24 @@ clt = Client(
 )
 ```
 
+For testing SSL endpoints you may wish to use a self-signed certificate.  If you do this and receive an `SSL: CERTIFICATE_VERIFY_FAILED` error message you might add the following configuration to your client:
+
+```python
+clt.config.verify_ssl= False
```
Comment (nopcoder) [inline docs/integrations/python.md:62]:
```suggestion
clt.config.verify_ssl = False
```

**Review by nopcoder — APPROVED** [review]
LGTM - minor comment in the version ssl code block

**Pull request merged**